### ISSUE TITLE
trailing-whitespace reports wrong column number

### ISSUE THREAD

**Issue by li-boxuan** [PyCQA/pylint#2000]
Currently, trailing-whitespace always reports column number 0. IMO it should be the position of the first trailing whitespace.

### Steps to reproduce
1. Add a file with trailing spaces, e.g. (note the trailing spaces)
```python
a = 1   
```
2. Run pylint on it

### Current behavior
C:  1, 0: Trailing whitespace (trailing-whitespace)

### Expected behavior
C:  1, 5: Trailing whitespace (trailing-whitespace)

### pylint --version output
Using config file /Users/liboxuan/pylint/pylintrc
pylint 2.0.0
astroid 1.6.3
Python 3.6.2 (default, Oct  1 2017, 03:29:21)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.37)]
